src/utils/bridge.js

```diff
@@ -288,33 +288,23 @@ export function logException(
  *
  * @param {number} timeoutMs Timeout in milliseconds after which to reject.
  *
- * @return {Promise<GBKitConfig>} Promise that resolves with GBKit config or rejects after timeout.
+ * @return {Promise<void>} Promise that resolves with GBKit config or rejects after timeout.
  */
 export function awaitGBKitGlobal( timeoutMs = 3000 ) {
 	return new Promise( ( resolve, reject ) => {
 		const startTime = Date.now();
 
 		const checkGBKit = () => {
 			if ( window.GBKit ) {
-				resolve( window.GBKit );
```

Comment:
The resolved values were never consumed. This utility is only intended for awaiting the readiness of the bridge-provided value.